apps/snmp_exporter/app.yaml

```diff
@@ -0,0 +1,71 @@
+{
+    "id": "/snmp_exporter",
+    "cmd": null,
+    "cpus": 1,
+    "mem": 1024,
+    "disk": 0,
+    "gpus": 0,
+    "instances": 1,
+    "executor": "",
+    "requirePorts": false,
+    "constraints": [
+        [
+            "secrets",
+            "LIKE",
+            "true"
```

Comment:
I know this pins things to the hypervisors but it might not be necessary either

Reply:
Are you referring to all three of these or just the last one?